storybook-docs/.storybook/main.ts

```diff
@@ -0,0 +1,55 @@
+/** @type { import('@storybook/react-vite').StorybookConfig } */
+import type { StorybookConfig } from '@storybook/react-vite';
+
+const config: StorybookConfig = {
+    stories: [
+        "../stories/**/*.mdx",
+        "../stories/**/*.stories.@(js|jsx|mjs|ts|tsx)",
+    ],
+    addons: [
+        "@storybook/addon-onboarding",
+        "@storybook/addon-links",
+        "@storybook/addon-essentials",
+        "@chromatic-com/storybook",
```

Comment:
what is this for?

Reply:
addon-essentails: contains functionalities such as controls, actions, etc.
addon-links: allow us to use inernal links to link to other stories. Will be usable in compoable components

addon-onboarding: I'll remove this!